src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,19 @@
+.catalog {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+
+  &__card {
+    width: 198px;
```

Comment:
your tests are failing because you have the wrong `width`. maybe it would be better to remove the class completely
![image](https://user-images.githubusercontent.com/22824431/235124776-31f38d95-fffa-42be-843c-83b867eec3d1.png)